conf/logback.xml

```diff
@@ -27,6 +27,7 @@
         <maxFileSize>2gb</maxFileSize>
       </timeBasedFileNamingAndTriggeringPolicy>
       <fileNamePattern>logs/metrics-portal.%d{yyyy-MM-dd_HH}.%i.log.gz</fileNamePattern>
+      <maxFileSize>2gb</maxFileSize>
```

Comment:
Ah, yeah. When the rolling policy was time and size, it was actually breaking.  Now that I merged your change and rebased, it's not required.